src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.styles.ts

```diff
@@ -100,3 +102,23 @@ export const propertyExpressionLabelStyles: Partial<ILabelStyles> = {
         paddingTop: 5
     }
 };
+
+export const getClearButtonStyles = memoizeFunction(
+    (theme: Theme, isTooltipVisible: boolean): Partial<IButtonStyles> => ({
+        root: {
+            color: theme.palette.themePrimary,
+            fontWeight: FontWeights.semibold,
+            textDecoration: 'underline',
+            fontSize: FontSizes.size14,
+            paddingBottom: 0,
+            paddingLeft: 0,
```

Comment:
can we fix it with margin instead of padding? This causes the focus state (when you tab) to be on top of the text. 
![image](https://user-images.githubusercontent.com/57726991/171731612-57b462df-89d5-435f-8904-f425c176a51d.png)

Reply:
Still doesn't look right on the most recent build. Do you see something else?
![image](https://user-images.githubusercontent.com/57726991/171748892-f7ba01af-9ae3-4220-be60-dd01781e44d6.png)

Reply:
The default padding for the button is what will give us the right focus border. If we can stick with the default it should be good out of the box.